mappings/net/minecraft/world/Heightmap.mapping

```diff
@@ -29,8 +29,10 @@ CLASS net/minecraft/class_2902 net/minecraft/world/Heightmap
 		ARG 2 z
 	METHOD method_16680 (Lnet/minecraft/class_2680;)Z
 		ARG 0 state
+	METHOD method_16681 getSuffocatesPredicate ()Ljava/util/function/Predicate;
```

Comment:
`method_16681` is a synthetic method. Have we agreed on mapping synthetic methods? Last time I saw about it was https://github.com/FabricMC/yarn/pull/1941#issuecomment-766391232.

Reply:
Mapping these don't hurt. We generate intermediary for these synthetics; somehow decompilers don't always remove them (they usually do for those synthetic ctors in comparison); on a side note, some synthetic methods are removed by decompilers, yet we can only see when we look up usages in enigma (since enigma doesn't think about handling synthetics when building up like method call index)